hadoop-common-project/hadoop-registry/src/main/java/org/apache/hadoop/registry/server/services/RegistryAdminService.java

```diff
@@ -224,7 +224,7 @@ protected void serviceStart() throws Exception {
           bindingDiagnosticDetails(),
           dumpRegistryRobustly(true));
 
-      LOG.error(" Failure {}", e, e);
+      LOG.error(" Failure {}", e.getMessage(), e);
```

Comment:
-1. to this change

L227 is valid; it calls e.toString on demand and handles any errors raised. the second e forms the stack trace.

*e.getMessage() is null on some exceptions, like NPE*

Reply:
I will revert it.